UI/index.html

```diff
@@ -39,7 +39,7 @@
                         </div>
                         <ul class="navigation__list">
                                 <li class="navigation__listitem">
-                                        <input type="text" class="navigation__listsearch">
+                                        <input type="text" class="navigation__listsearch" placeholder="search for meetups">
                                         <a href="#" class="navigation__listlink">
                                         <span class="navigation__listsearch-btn">Search</span>
                                         </a>
```

Comment:
Please use two character spacing in all files.

Reply:
okay
thanks